src/app/table/table.component.html

```diff
@@ -16,7 +16,7 @@
       <th mat-header-cell *matHeaderCellDef></th>
       <td mat-cell *matCellDef="let element" [id]="rowActionIcon" (click)="emitRowAction(element)">
         <button mat-button>
-          <mat-icon>remove_shopping_cart</mat-icon>
+          <mat-icon>{{rowActionIcon}}</mat-icon>
```

Comment:
@timothyr thank you! Please set the default value to this variable in table.component.ts, so that the example components in this project have an icon shown

Reply:
The default should remain null, because in the example the Customers table contains no row action icon. If I change the default value in table.component.ts then an icon shows up, even though an icon wasn't specified for the table.

This is because the icon uses ngIf on rowActionIcon here:

```
    <!-- action column -->
    <ng-container *ngIf="rowActionIcon?.length" [matColumnDef]="rowActionIcon">
      <th mat-header-cell *matHeaderCellDef></th>
      <td mat-cell *matCellDef="let element" [id]="rowActionIcon" (click)="emitRowAction(element)">
        <button mat-button>
          <mat-icon>{{rowActionIcon}}</mat-icon>
        </button>
      </td>
    </ng-container>
```

With the change I made, both example tables are working as intended and if someone changes the icon in orders.component.html then it uses the specified icon instead of the hardcoded shopping cart icon.